lib/usd/translators/meshWriter_BlendShapes.cpp

```diff
@@ -672,6 +671,17 @@ MObject PxrUsdTranslators_MeshWriter::writeBlendShapeData(UsdGeomMesh& primSchem
                     MString                   curTargetNameMStr;
                     MString                   curTargetLongNameMStr;
                     if (!targetMesh.isNull()) {
+                        MFnDagNode dagNode(targetMesh);
+                        MString    nodeName;
+                        if (dagNode.parentCount() > 0) {
+                            MFnDagNode parentDagNode(dagNode.parent(0));
+                            curTargetNameMStr
+                                = UsdMayaUtil::GetUniqueNameOfDagNode(parentDagNode.object());
+                            curTargetLongNameMStr = MString(curTargetNameMStr);
```

Comment:
IDK why there is an explicit construction/cast of a MString here, but the old code was doing it too, so there is little harm.

Reply:
I used them because they were there before, but I can remove them as well, no problem